locales/fr/base.json

```diff
@@ -152,14 +182,14 @@
     "delete": "Supprimer",
     "move-to-group-custom": "Déplacer vers {{title}}",
     "empty-trash": "Vider la corbeille",
-    "empty-trash-question": "Êtes-vous sûr de vouloir vider la corbeille ?",
-    "title-asc": "Titre : Croissant",
-    "title-desc": "Titre : Décroissant"
+    "empty-trash-question": "Êtes-vous sûr de vouloir vider la corbeille ?",
```

Comment:
Why is there space added before `:` and `?`?

Reply:
Hi Sallar,

the unbreakable space before double punctuations signs is a typographic rule in French.

Kind regards

Reply:
@fmommeja excuse my ignorance then! thanks :)

Reply:
@sallar no problem.
I guess rules are specific to each and every language… you can't (neither do I) know everything of it.  ;)